specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
```

Comment:
nit: either this should be indented under Instrumentation Libraries or Instrumentation Libraries should further define what is in scope -- I assume it is applications and serverless.

Reply:
Good point - I'll put that under the Istrumentation Libraries section. Thanks!